sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/FeatureFlagConfigurationSetting.java

```diff
@@ -45,6 +46,7 @@ public FeatureFlagConfigurationSetting(String featureId, boolean isEnabled) {
         this.isEnabled = isEnabled;
         super.setKey(KEY_PREFIX + featureId);
         super.setContentType(FEATURE_FLAG_CONTENT_TYPE);
+        clientFilters = new ArrayList<>();
```

Comment:
What happens service side if it receives an empty `clientFilters` list? The other option would be to defer this instantiation and only perform it when `addFilter` is called and `clientFilters == null`